types/es-to-primitive/es2015.d.ts

```diff
@@ -0,0 +1,5 @@
+declare function ToPrimitive(
+	input: any,
+	hint?: Function
+): string | number | boolean | symbol | null | undefined;
```

Comment:
```suggestion
): string | number | bigint | boolean | symbol | null | undefined;
```

Reply:
I’ve applied this in **TypeScript** 3.2 specific types.